include/boost/geometry/formulas/area_formulas.hpp

```diff
@@ -472,43 +487,46 @@ class area_formulas
 
         CT excess;
 
-        auto const half_pi = math::pi<CT>() / 2;
-        bool meridian = lon2r - lon1r == CT(0)
-            || lat1r == half_pi || lat1r == -half_pi
-            || lat2r == half_pi || lat2r == -half_pi;
+        CT lon12r = lon2r - lon1r;
+        math::normalize_longitude<radian, CT>(lon12r);
```

Comment:
-> `const`

Reply:
`lon12r` cannot be `const` since `normalize_longitude` is changing it.

Reply:
All right, missed that, thanks